generators/pattern-library/render.js

```diff
@@ -29,46 +39,22 @@ module.exports = function ( config ) {
 
 function globPartials( config ) {
 
-    var partials = config.settings.template.partials.map( function ( dir ) {
-
-        return globber({
-            src: [ path.join( dir.src, '**/*' ) ],
-            options: dir.options
-        })
-        .then( function ( files ) {
-
-            return files.reduce( function ( collection, filePath ) {
-
-                collection.push({
-                    cwd: dir.src,
-                    file: filePath,
-                    name: path.relative( dir.src, filePath ).replace( path.parse( filePath ).ext, '' )
-                });
-
-                return collection;
-
-            }, [] );
-        });
-    });
-
-    return Promise.all( partials )
-    .then( function ( files ) {
-
-        // files = _.flatten( files );
-        config.settings.template.partials = _.flatten( files );
+    return globFiles( config.settings.template.partials )
+    .then( function ( partials ) {
```

Comment:
If we are going to start using arrow functions, shouldn't all new anonymous functions use the arrow function to keep with consistency?